solver/llbsolver/mounts/mount.go

```diff
@@ -50,7 +50,7 @@ func (mm *MountManager) getRefCacheDir(ctx context.Context, ref cache.ImmutableR
 		cacheMounts:     mm.cacheMounts,
 		cm:              mm.cm,
 		globalCacheRefs: sharedCacheRefs,
-		name:            fmt.Sprintf("cached mount %s from %s", m.Dest, mm.managerName),
+		name:            fmt.Sprintf("cached mount %s from %s with id %q", m.Dest, mm.managerName, id),
```

Comment:
@tonistiigi before I press merge, just want to confirm this didn't get lost in the PR. It's technically not a new log, it's changing the name you see when you call disk usage to also include the actual id of the cache mount as opposed to just its mount location.

I figured there's no expectation of stability when it comes to these names, so this should be fine, just wanted to double check.

Reply:
sgtm

Reply:
Only thing is that I believe by default ID and `dest` are the same value. If that is the case would be nice to show ID only if it is different.

Reply:
Oh sure, I forgot that's the dockerfile behavior, will fix before merge